go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/fogleman/gg
+
+go 1.22
```

Comment:
`1.22` this version has not been released yet.

Reply:
It has now. :)